modules/021-cni-cilium/templates/agent/daemonset.yaml

```diff
@@ -458,3 +461,8 @@ spec:
                 path: server.crt
               - key: tls.key
                 path: server.key
+{{- end  }}
+
+{{ $agent_daemonset_generation := include "agent_daemonset_template" . | sha256sum }}
+
+{{- include "agent_daemonset_template" . | replace "%%DS_GENERATION%%" $agent_daemonset_generation }}
```

Comment:
please do not use replace, use template variables

Reply:
@AndreyPavlovFlant need to add $context var and replace all dots to $context in template.